### ISSUE TITLE
Get SPCascadeDropdowns Working with SharePoint 2013 Quick Edit View

### ISSUE THREAD

**Issue by Johesmil** [sympmarc/SPServices#59]
I took the liberty to add this issue from codeplex as it was not only the first, but also the second highest voted issue there. 

**From [Get SPCascadeDropdowns Working with SharePoint 2013 Quick Edit View]_(http://spservices.codeplex.com/workitem/10181)**
"SPCascadeDropdowns does not work with the "Quick Edit" view - aka the new Datasheet view - in SharePoint 2013 as of SPServices 2013.01."_

**From [SP Cascading Drop Downs in Quick Edit mode](http://spservices.codeplex.com/workitem/10286)**
"From https://spservices.codeplex.com/discussions/567707 

Marc - Great stuff! 

Is there a way to have the cascade feature work even when a user is editing their list in "Quick Edit" mode (datasheet entry in SP 2013)? 
After I select my first column (parent), ALL the choices are still available in the second (child) column; no filtering has occurred. I know I placed the code on the New and Edit form... 

Can this be done? 
Thanks, Jean"

**Comment by sympmarc** [comment]
This would be very hard to do, though I'm not sure it's impossible. Because of this, I've never tried to get it working. These days, relying on Microsoft not changing the DOM is a risky business.

**Comment by techiechickie** [comment]
I realize this is an older post, are there any new coding options available for getting the cascade to work in Quick Edit or Datasheet View in Sharepoint Online?
Thank you!

**Comment by sympmarc** [comment]
@techiechickie No, those views are complex in the UI. The Datasheet view - in particular - uses an ActiveX control, which doesn't give us an HTML surface to work with. Modern SharePoint pages don't allow arbitrary script (no CEWP anymore) as the architecture has completely changed.

Your best bet is to use PowerApps.
